trin-core/src/utp/stream.rs

```diff
@@ -274,20 +274,14 @@ impl UtpListener {
 
                 match self.listening.get(&conn.receiver_connection_id) {
                     Some(message_type) => match message_type {
-                        UtpMessageId::OfferAcceptStream => {
-                            match UtpAccept::from_ssz_bytes(&received_stream[..]) {
-                                Ok(payload) => {
-                                    for (key, content) in payload.message {
-                                        // TODO: Implement this with overlay store
-                                        debug!("Store {key:?}, {content:?}");
-                                    }
-                                }
-                                Err(_) => debug!("Recv malformed data on handing UtpAccept"),
-                            }
+                        UtpMessageId::AcceptStream(content_keys) => {
+                            // TODO: Implement this with overlay store and decode receiver stream if multiple content values are send
+                            debug!("Store {content_keys:?}, {received_stream:?}");
+                        }
+                        UtpMessageId::FindContentData(_content) => {
```

Comment:
Wouldn't content data via uTP stream correspond to `UtpMessageId::FindContentStream`? Maybe I misunderstand the difference between the two variants.

Reply:
This is kinda tricky because when we respond with the `Content` message with `connection-id`,  we want to keep somewhere the actual content data that we want to stream when we receive `SYN` packet from the requester (the node which sent `FindContent` rpc). 

So right now we keep this data in memory as part of the `UtpMessageId` as `FindContentData(Content)` and we store it on the `send_send_id` channel only, we keep `FindContentStream` for `conn_recv_id`: https://github.com/ethereum/trin/blob/7f66b18221c1124a7e386c193feed5a8a132a0fd/trin-core/src/portalnet/overlay_service.rs#L626